baseapp/queryrouter.go

```diff
@@ -0,0 +1,51 @@
+package baseapp
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// QueryRouter provides queryables for each query path.
+type QueryRouter interface {
+	AddRoute(r string, h sdk.Querier) (rtr QueryRouter)
```

Comment:
h - handler, right?

what's `rtr`?

Reply:
I'd change `r` --> `route` and `h` --> `handler`

Reply:
This is fine IMO